python/py-hyrule/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-hyrule
+version             0.3.0
+categories-append   devel
+license             MIT
+supported_archs     noarch
+maintainers         nomaintainer
+description         A utility library for the Hy programming language
+long_description    Hyrule is a utility library for the Hy programming \
+                    language. It can be thought of as the Hy equivalent, or addition, \
+                    to Python’s standard library. While intended primarily for Hy \
+                    programs, its functions and classes can be used in Python as with \
+                    any other Python library\; just import hyrule. Hyrule’s macros, on \
+                    the other hand, are only really usable in Hy.
+
+homepage            https://hyrule.readthedocs.io/
+
+checksums           rmd160 635180f049af1e3dd8e0ed0d6da247ed476b8ff1 \
+                    sha256 b65127d94d71aa7c05e7eb1c3e550b389247c66f41af24c73f1fe743d39410b4  \
+                    size   45291
+
+python.versions     38 39 310 311
+
+python.pep517       no
+
+if {${name} ne ${subport}} {
+    depends_build-append    port:py${python.version}-setuptools \
+                            port:py${python.version}-wheel
+
+    depends_lib-append      port:py${python.version}-hy
+
+    livecheck.type          none
+} else {
+    livecheck.type          pypi
+}
```

Comment:
the `livecheck` stuff here is as it is set by the `python` PG as default so that can be removed.